File: Fibonacci.java
/**
 * Fibonacci performance test
 *
 * @author Stoyan Rangelov (devsoul) <stoyan.rangelov@gmail.com>
 */

public class Fibonacci {

    public static int fib(int n) {
        if (n < 2) {
            return n;
        }
        return fib(n - 1) + fib(n - 2);

    }

    public static void main(String[] args) {
        System.out.print(fib(30) + "\n");
    }
}

File: fibonacci.py
# Fibonacci performance test
# Author Stoyan Rangelov (devsoul) <stoyan.rangelov@gmail.com>

def fib(n):
    if n < 2:
        return n
    else:
        return fib(n-1) + fib(n-2)

print(fib(30))
